api-report/container-loader.api.md

```diff
@@ -182,8 +182,7 @@ export class DeltaManager extends TypedEventEmitter<IDeltaManagerInternalEvents>
     dispose(): void;
     // (undocumented)
     get disposed(): boolean;
-    // (undocumented)
```

Comment:
We should question every change to api.md, and see if API being changed actually belongs here. I believe the answer here is no.
It would be great to remove ./deltamanager from index.ts and instead export (by moving into other files like utils.ts) anything that truly needs to be exported from this package.

Reply:
#6640